comictaggerlib/ui/settingswindow.ui

```diff
@@ -450,99 +545,344 @@
-               </property>
+              <widget class="QWidget" name="tAdd" native="true">
+               <attribute name="title">
+                <string>Add Missing</string>
+               </attribute>
+               <layout class="QVBoxLayout" name="verticalLayout_9">
+                <property name="spacing">
+                 <number>0</number>
+                </property>
+                <property name="leftMargin">
+                 <number>0</number>
+                </property>
+                <property name="topMargin">
+                 <number>0</number>
+                </property>
+                <property name="rightMargin">
+                 <number>0</number>
+                </property>
+                <property name="bottomMargin">
+                 <number>0</number>
+                </property>
+                <item>
+                 <widget class="QPlainTextEdit" name="addTextEdit">
+                  <property name="plainText">
+                   <string>Metadata missing from the original is added from the the new metadata.
+
+Example:
+(Series=batman, Issue=1, Tags=[batman,joker,robin])
++ (Series=Superman, Issue=10, Publisher=DC Comics, Tags=[superman,lois lane,lex luthor])
+= (Series=batman, Issue=1, Publisher=DC Comics, Tags=[batman,joker,robin])</string>
+                  </property>
+                  <property name="textInteractionFlags">
+                   <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
+                  </property>
+                 </widget>
+                </item>
+               </layout>
               </widget>
-             </item>
-             <item row="4" column="0">
-              <widget class="QCheckBox" name="cbxCopyStoryArcsToTags">
-               <property name="text">
-                <string>Copy Story Arcs to Generic Tags</string>
-               </property>
+              <widget class="QWidget" name="tCombine">
+               <attribute name="title">
+                <string>Combine</string>
+               </attribute>
+               <layout class="QVBoxLayout" name="verticalLayout_10">
+                <property name="spacing">
+                 <number>0</number>
+                </property>
+                <property name="leftMargin">
+                 <number>0</number>
+                </property>
+                <property name="topMargin">
+                 <number>0</number>
+                </property>
+                <property name="rightMargin">
+                 <number>0</number>
+                </property>
+                <property name="bottomMargin">
+                 <number>0</number>
+                </property>
+                <item>
+                 <widget class="QPlainTextEdit" name="combineTextEdit">
+                  <property name="plainText">
+                   <string>Same as Overlay except lists (tags, characters, genres, credits, etc.) are combined.
```

Comment:
This is not accurate with the current code anymore

Reply:
How's this:
```
Combines lists (tags, characters, genres, credits, etc.) replacing any duplicates with the new record. All other non-empty values replace any current.
```

Reply:
Is there a reason for the code to not match the description?

Reply:
What am I missing?
```
Combines lists (tags, characters, genres, credits, etc.) replacing any duplicates with the new record. All other non-empty values replace any current.
```
I think that is correct? "replacing" maybe isn't the perfect word, "using the new value with any duplicate items"?

Reply:
I'm with you now. It was a 50/50; change the description or change the code and I chose poorly.

If the command line empty string is taken out (presuming that command line metadata will always be used in `overlay` mode) then they look to do the same outside of lists/sets?
```python
    def assign_overlay(self, cur: Any, new: Any) -> Any:
        """Overlay - When the 'new' object has non-None values, overwrite 'cur'(rent) with 'new'."""
        if new is None:
            return cur
        # An empty string can be used to clear a list or set via the commandline
        if isinstance(new, str) and len(new) == 0:
            if isinstance(cur, (list, set)):
                cur.clear()
                return cur
            else:
                return None
        elif isinstance(new, (list, set)) and len(new) == 0:
            return cur
        else:
            return new
```
```python
    def assign_combine(self, cur: Any, new: Any) -> Any:
        """Combine - Combine lists and sets. All non-None values from new replace cur(rent)"""
        if new is None:
            return cur
        if isinstance(new, (list, set)) and isinstance(cur, (list, set)):
            if len(new) > 0 and isinstance(new, list) and isinstance(new[0], Url):
                return list(set(new).union(cur))
            return self.assign_dedupe(new, cur)
        else:
            return new
```
Apologies if I'm missing the obvious.